src/declaration.d.ts

```diff
@@ -0,0 +1 @@
+declare module "*.png"
```

Comment:
This is a declaration file for typescripts. which helps with many things, like creating custom types for libraries that don't have types themselves. Or in this case we are declaring a module, any file that ends with `.png`, so that typescripts would stop complaining about it in the get-icon helper files